PyInstaller/compat.py

```diff
@@ -91,7 +91,10 @@
 
 # Linux distributions such as Alpine or OpenWRT use musl as their libc implementation and resultantly need specially
 # compiled bootloaders. On musl systems, ldd with no arguments prints 'musl' and its version.
-is_musl = is_linux and "musl" in subprocess.getoutput(["ldd"])
+if sys.version_info > (3, 11):
+    is_musl = is_linux and "musl" in subprocess.getoutput(["ldd"], encoding="utf-8")
+else:
+    is_musl = is_linux and "musl" in subprocess.getoutput(["ldd"])
```

Comment:
Can you use `is_py312` – makes it easier to clean up dead code when we drop support for old Python versions?

Or possibly just use `subprocess.run()` (make sure you set `stderr=subproccess.STDOUT`).

Reply:
Thanks!